src/api/bank-accounts/bank-account-requests.md

```diff
@@ -41,10 +41,10 @@ may be stored against those Bank Accounts.
 
 ### Bank Account Request Types
 
-|      Name      |                                      description                                       |
-| :------------- | :------------------------------------------------------------------------------------- |
-| quartz-details | Request will initiate an authorization flow for authorizing access to account details. |
-| quartz-payment | Request will initiate an authorization flow for authorizing access to create payments. |
+|      Name       |                                      description                                       |
+| :-------------- | :------------------------------------------------------------------------------------- |
+| account-consent | Request will initiate an authorization flow for authorizing access to account details. |
```

Comment:
Why's this not quartz-accounts anymore?

Reply:
Changed it in the wrong file - have updated 🫠